common/persistence/persistence-tests/clusterMetadataManagerTest.go

```diff
@@ -98,20 +99,27 @@ func (s *ClusterMetadataManagerSuite) TestClusterMembershipUpsertCanReadAny() {
 
 // TestClusterMembershipUpsertCanRead verifies that we can UpsertClusterMembership and read our result
 func (s *ClusterMetadataManagerSuite) TestClusterMembershipUpsertCanPageRead() {
-	hostID := []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}
+	// Expire previous records
+	// Todo: MetaMgr should provide api to clear all members
+	time.Sleep(time.Second * 2)
```

Comment:
[Nit] Can this be part of the Suite test setup/teardown function?

Reply:
Once that functionality exists, yes definitely. We have a similar problem in other areas where tests in the same suites rely on cleanup of the tests rather than enforcing it themselves.